packages/yoroi-extension/app/api/ada/index.js

```diff
@@ -1377,23 +1371,28 @@ export default class AdaApi {
         false,
       );
 
+      const defaultToken = {
+        defaultNetworkId: wallet.networkId,
+        defaultIdentifier: wallet.defaultTokenId,
+      };
```

Comment:
Might be a good idea to introduce `wallet.defaultToken`, since there's no variable state in it